Equinox/SimpleTimer.h

```diff
@@ -28,35 +28,11 @@ class SimpleTimer
 		return pausedTicks;
 	}
 
-	void Resume()
-	{
-		if(!running)
-		{
-			running = true;
-			runTicks = SDL_GetTicks() - pausedTicks;
-			pausedTicks = 0;
-		}
-	}
-
-	void Clear()
-	{
-		running = false;
-		runTicks = 0;
-		pausedTicks = 0;
-	}
-
 	unsigned int Read() const
```

Comment:
Read now is not correctly implemented, it must always return the elapsed time if running, or the elapsed time when until the timer was stopped. Please check the complex timer implementation: https://github.com/NeebulaGames/EquinoxEngine/blob/feature/testing-timers/Equinox/ComplexTimer.h